esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -188,6 +188,14 @@ void CWsDfuEx::init(IPropertyTree *cfg, const char *process, const char *service
     maxFileAccessExpirySeconds = serviceTree->getPropInt("@maxFileAccessExpirySeconds", defaultMaxFileAccessExpirySeconds);
 }
 
+StringBuffer &createAuditUserString(IEspContext &context, const char *userID, StringBuffer &auditUser)
+{
+    if (!isEmptyString(userID))
```

Comment:
for consistency's sake, can we use "ANONYMOUS" or some other value for missing userID